admin/server-timing.php

```diff
@@ -58,25 +99,23 @@ static function() {
 					),
 					array( 'code' => array() )
 				);
-				?>
-				<br>
-				<?php
```

Comment:
+1 to remove this, but it needs to be replaced with a space (`echo ' ';`), otherwise the text formatting is off.

Reply:
Oh yes, good catch. Fixed in 60005c4